packages/lib/actionClass/service.ts

```diff
@@ -116,8 +125,10 @@ export const createActionClass = async (
       select,
     });
 
-    // revalidate cache
-    revalidateTag(getActionClassesCacheTag(environmentId));
+    actionClassCache.revalidate({
```

Comment:
we also need to revalidate by `name`, right?

Reply:
done